cmd/antrea-agent/agent.go

```diff
@@ -922,6 +924,17 @@ func run(o *Options) error {
 		go flowExporter.Run(stopCh)
 	}
 
+	// Start the node latency monitor.
+	if features.DefaultFeatureGate.Enabled(features.NodeLatencyMonitor) {
+		nodeLatencyMonitor := monitortool.NewNodeLatencyMonitor(
+			nodeInformer,
+			nodeLatencyMonitorInformer,
+			nodeConfig,
+			networkConfig.TrafficEncapMode,
+		)
+		go nodeLatencyMonitor.Run(stopCh)
+	}
```

Comment:
I took some time to think about it, and this should probably be gated by `if o.nodeType == config.K8sNode`. Check out the rest of this file for reference.

Reply:
OK, I add the `nodeType` check.